bridges/WallpaperflareBridge.php

```diff
@@ -0,0 +1,40 @@
+<?php
+class WallpaperflareBridge extends XPathAbstract {
+	const NAME = 'Wallpaperflare';
+	const URI = 'https://wallpaperflare.com';
+	const DESCRIPTION = 'Wallpaperflare is a provider for Wallpapers on nearly every topic, especially for Anime';
+	const MAINTAINER = 'dhuschde';
+	const PARAMETERS = array(
+		'' => array(
+			'search' => array(
+				'name' => 'Search',
+				'required' => true
+		)
+	));
+	const CACHE_TIMEOUT = 3600; //1 hour
+	const XPATH_EXPRESSION_ITEM = './/figure';
+	const XPATH_EXPRESSION_ITEM_TITLE = './/img/@title';
+	const XPATH_EXPRESSION_ITEM_CONTENT = '';
+	const XPATH_EXPRESSION_ITEM_URI = './/a[@itemprop="url"]/@href';
+	const XPATH_EXPRESSION_ITEM_AUTHOR = '/html[1]/body[1]/main[1]/section[1]/h1[1]';
+	const XPATH_EXPRESSION_ITEM_TIMESTAMP = '';
+	const XPATH_EXPRESSION_ITEM_ENCLOSURES = './/img/@data-src';
+	const XPATH_EXPRESSION_ITEM_CATEGORIES = './/figcaption[@itemprop="caption description"]';
+	const SETTING_FIX_ENCODING = false;
+
+	protected function getSourceUrl(){
+		$search = $this->getInput('search'); return 'https://www.wallpaperflare.com/search?wallpaper=' . $search;
+	}
+
+	public function getIcon() {
+		$feedicon = 'https://www.google.com/s2/favicons?domain=wallpaperflare.com/'; return $feedicon;
+	}
+
+	public function getName() {
+		if(!is_null($this->getInput('search'))) {
+			$search = $this->getInput('search'); return 'Wallpaperflare - ' . $search;
+			} else {
+			return 'Wallpaperflare';
```

Comment:
Use can use parent::getName()

Reply:
I don't know what you meant by that...

Reply:
I think you meant, that I should use the Name of the Webpage, but that isn't a very nice name...